### PR TITLE
Replace voting power with balances in Voting terminal

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Replace balance with voting power

Task: [APP-2368](https://aragonassociation.atlassian.net/browse/APP-2368)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2368]: https://aragonassociation.atlassian.net/browse/APP-2368?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/votingTerminal/index.tsx [hunk 1]
```diff
@@ -105,15 +110,26 @@ export const VotingTerminal: React.FC<VotingTerminalProps> = ({
...
+          if (daoToken?.address && wallet.address) {
+            balance = await fetchPastVotingPower({
+              tokenAddress: daoToken.address as string,
+              address: wallet.address as string,
+              blockNumber: blockNumber as number,
+            });
+          }
           return {
             ...voter,
+            tokenAmount: balance ? formatUnits(balance, 18) : voter.tokenAmount,
```
Comment (cgero-eth) [inline src/containers/votingTerminal/index.tsx]: We probably need to use the decimals set on the daoToken variable:

```suggestion
            tokenAmount: balance ? formatUnits(balance, daoToken.decimals) : voter.tokenAmount,
```

File: src/pages/proposal.tsx [hunk 2]
```diff
@@ -690,6 +690,8 @@ export const Proposal: React.FC = () => {
 
           <VotingTerminal
             status={proposal.status}
+            daoToken={daoToken}
+            blockNumber={proposal?.creationBlockNumber as number}
```
Comment (cgero-eth) [inline src/pages/proposal.tsx]: The cast to `number` is probably redundant here
```suggestion
            blockNumber={proposal?.creationBlockNumber}
```

**Review by cgero-eth — COMMENTED** [review]
Thanks for the changes @sepehr2github, left two small comments.

**Review by cgero-eth — DISMISSED** [review]

**Comment by RuggeroCino** [comment]
@sepehr2github the pipeline is currently failing

**Review by cgero-eth — COMMENTED** [review]
PR looks good but there are some linting issues and the pipeline is failing

**Review by cgero-eth — APPROVED** [review]

**Pull request merged**